tests/kernel/mem_protect/userspace/testcase.yaml

```diff
@@ -14,6 +14,11 @@ tests:
       - CONFIG_COMMON_LIBC_MALLOC_ARENA_SIZE=0
     platform_exclude:
       - ucans32k1sic
+      # Commit cb19bd4fc2541931beb0e1f157127f7a50a082d4
+      # explicitly disabled CONFIG_USERSPACE so xenvm
+      # boards cannot run this test.
```

Comment:
That commit says that it "temporarily" disabled userspace for these boards, but it merged almost two years ago.

Seems like a better solution here would be to disable ARCH_HAS_USERSPACE for the boards in question and then file a bug to get this fixed?

Reply:
there is another PR (https://github.com/zephyrproject-rtos/zephyr/pull/70129)  dealing with this, CONFIG_USERSPACE=n in a board config has no effect anyways.

Reply:
Removed the commit as #70129 has fixed build error.